src/pages/EditRequestPage.js

```diff
@@ -56,6 +60,15 @@ const propTypes = {
         email: PropTypes.string,
     }),
 
+    /** Collection of tags attached to a policy */
+    policyTags: PropTypes.objectOf(
+        PropTypes.shape({
+            name: PropTypes.string,
+            required: PropTypes.bool,
+            tags: PropTypes.objectOf(tagPropTypes),
+        }),
```

Comment:
I see this `shape` used across components. We can extract it to a separate file to re-use. Like for categories: https://github.com/Expensify/App/blob/main/src/components/categoryPropTypes.js.